packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/DiffLegendPaletteDropdown.module.css

```diff
@@ -31,6 +31,7 @@
 
 .dropdownItem svg {
   margin-left: 1em;
+  fill: white;
```

Comment:
didn't we move all colors to a single file ? should this be a variable instead to support light / dark mode?